app/src/sharedTest/java/org/oppia/app/walkthrough/WalkthroughFragment.kt

```diff
@@ -0,0 +1,42 @@
+package org.oppia.app.walkthrough
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.*
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.utility.ProgressMatcher.withProgress
+
+/** Tests for [WalkthroughFragment]. */
+@RunWith(AndroidJUnit4::class)
+class OnboardingFragmentTest {
+
+  @Test
+  fun testWalkthroughFragment_skipNextButtonIsVisible() {
```

Comment:
Normal convention for any test case:
_`testAction_withOneCondition_withSecondCondition_hasExpectedOutcome`_

Reply:
done